### PR TITLE
Add vim swap files to gitignore and a Euler's number approximation benchmark

### PR DESCRIPTION
**Pull request by keikun555**

### CODE REVIEW

**Comment by keikun555** [comment]
Forgot to make changes to the benchmark documentation, let me add that

**Comment by keikun555** [comment]
Done

**Review by sampsyo — COMMENTED** [review]
Very nice! Here's just a minor suggestion for the `.gitignore` change. (The benchmark looks fine!)

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,7 @@ __pycache__
 .vscode
 
 **/*.o
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# vim swap files
+*.sw*
```
[the remark this answers is not in the document]
Comment (keikun555) [reply .gitignore]: Taken in the suggestions

**Comment by sampsyo** [comment]
Looks good; thanks!!

**Pull request merged**